includes/Order/functions.php

```diff
@@ -44,30 +44,49 @@ function dokan_get_seller_amount_from_order( $order_id, $get_array = false ) {
  *
  * @global object $wpdb
  * @param int $seller_id
- * @param string $status
- * @param string $order_date
- * @param int $limit
- * @param int $offset
- * @param int $customer_id
+ * @param array $args
  *
  * @return array
  */
-function dokan_get_seller_orders( $seller_id, $status = 'all', $order_date = null, $limit = 10, $offset = 0, $customer_id = null ) {
-    // get all function arguments as key => value pairs
-    $args = get_defined_vars();
-
+function dokan_get_seller_orders( $seller_id, $args ) {
     global $wpdb;
 
+    $defaults = [
+        'status'      => 'all',
+        'order_date'  => null,
+        'limit'       => 10,
+        'offset'      => 0,
+        'customer_id' => null,
+        'total_high'  => '',
+        'total_low'   => '',
+        'start_date'  => '',
+        'end_date'    => '',
+    ];
+
+    $args = wp_parse_args( $args, $defaults );
+
+    $args['seller_id'] = $seller_id;
+
+    $status      = $args['status'];
+    $order_date  = $args['order_date'];
+    $limit       = $args['limit'];
+    $offset      = $args['offset'];
+    $customer_id = $args['customer_id'];
+    $total_high  = $args['total_high'];
+    $total_low   = $args['total_low'];
+    $start_date  = $args['start_date'];
+    $end_date    = $args['end_date'];
```

Comment:
Why do you need to store all these data in seperate variables? Avoid all these unnecessary declaration and directly use `$args` where needed. Like instead of `$status`, use `$args['status']`